lib/connection.js

```diff
@@ -230,6 +230,10 @@ var Connection = function (options, container) {
     this.closed_with_non_fatal_error = false;
 
     for (var i in aliases) {
```

Comment:
Perhaps aliases.forEach() might be cleaner here?

Reply:
yeah, and there is no need to check for hasOwnProperty in this case
I'll send an update now